translations/pt-br/04-ferramentas-utilizadas.md

```diff
@@ -0,0 +1,142 @@
+- **Controle de versão distribuído**: Ao contrário dos sistemas de controlo de versão centralizados, o Git é um sistema de controlo de versão distribuído. Isso significa que cada desenvolvedor tem uma cópia completa do repositório na máquina local, permitindo-lhes trabalhar offline e independentemente de um servidor central.
+
+### Introdução ao GitHub
+
+O GitHub é uma plataforma baseada na web para controlo de versão e colaboração que utiliza o Git como tecnologia subjacente. Ele fornece uma interface simples e amigável para gerir repositórios Git, bem como uma série de recursos e ferramentas para colaborar com outros desenvolvedores.
+
+Algumas das principais funcionalidades do GitHub incluem:
+
+- **Rastramento de issues**: O GitHub fornece um sistema integrado de rastreamento de problemas que permite que os desenvolvedores criem, atribuam e gerenciem problemas relacionados aos seus projetos. Os problemas podem ser categorizados usando etiquetas, marcos e pessoas atribuídas, tornando fácil acompanhar o progresso do projeto e priorizar tarefas.
+- **Pull requests**: O GitHub introduz o conceito de pedidos de pull, que permitem que os desenvolvedores proponham alterações à base de código de um projeto. Os pedidos de pull permitem a revisão de código, discussão e colaboração antes que as alterações sejam mescladas na base de código principal.
+- **Forking**: O GitHub permite que os utilizadores criem uma cópia de um repositório, conhecida como fork. Isso permite que eles façam alterações na base de código sem afetar o projeto original. Uma vez que as alterações estejam concluídas, eles podem enviar um pedido de pull para que as suas alterações sejam revisadas e potencialmente mescladas na base de código principal.
+- **GitHub Actions**: O GitHub Actions é uma poderosa ferramenta de automação que permite criar fluxos de trabalho personalizados para automatizar tarefas como a construção, teste e implantação do seu código.
+
+### Começando com Git e GitHub
+
+Para começar com Git e GitHub, siga estes passos:
+
+1. **Instalar o Git**: Baixe e instale o Git no seu computador visitando o site oficial do Git (https://git-scm.com/) e seguindo as instruções de instalação para o seu sistema operacional.
+2. **Configurar o Git**: Após instalar o Git, abra um terminal ou prompt de comando e configure as informações do seu usuário Git executando os seguintes comandos:
+
+```bash
+git config --global user.name "Seu nome"
+git config --global user.email "seu.email@exemplo.com"
+```
+
+Substitua "Seu nome" pelo seu nome completo e "seu.email@exemplo.com" pelo endereço de e-mail que você usou para se inscrever no GitHub.
+
+#### Praticando Contribuições com o VS Code
+
+Para praticar o uso do GitHub, vamos passar por criar um repositório, clonar o repositório, fazer alterações e empurrar essas alterações.
```

Comment:
```suggestion
Para praticar o uso do GitHub, veremos como criar um repositório, clonar o repositório, fazer alterações e empurrar essas alterações.
```
Well, I don't like the repetition, but the source is as it is. "repositório" and "alterações"